ur5_example.R

```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers
+ur5=read.csv(file="ur5testresult_halfspeed_payload1.6lb_1 (1).csv",header=FALSE)

```

Comment:
From: https://google.github.io/styleguide/Rguide.xml

"Spacing
Place spaces around all binary operators (=, +, -, <-, etc.). "

To automatically enforce R style convention, please download a "linter", e.g. https://cran.r-project.org/web/packages/lintr/README.html